network/gossip/libp2p/libp2pNode_test.go

```diff
@@ -544,3 +579,57 @@ func generateNetworkingKey(seed string) (crypto.PrivKey, error) {
 	prvKey, _, err := crypto.GenerateKeyPairWithReader(crypto.ECDSA, 0, r)
 	return prvKey, err
 }
+
+// newSilentNode returns a TCP listener and a node which never replies
+func newSilentNode(t *testing.T) (net.Listener, NodeAddress) {
```

Comment:
copied over from one of libp2p's test - https://github.com/libp2p/go-libp2p-swarm/blob/master/dial_test.go#L154